### ISSUE TITLE
[Question/Regression?] Where/How do i mass set customreactions in the bot now?

### ISSUE THREAD

**Issue by DrKittens** [Kwoth/NadekoBot#348]
Current Version the updater served me is: NadekoBot v0.9.6010.24597

Last version i was using customreactions went inside config.json
They now do not go there and there is a command i need to use to re-add all 400 custom reactions i had setup.

Where are the custom reactions now stored and can i just re-add my existing json formatted reactions to it, or do i really have to add every single reaction to the bot using the user-friendly method of typing ".acr "%mention% keyword" Stuff Here" 400+ times???

Rolling back a version til i get a reply :/
Basically im looking for an answer to "how do i mass import existing customreactions into the latest served version"

**Comment by Kwoth** [comment]
They still go there. Just copy paste your old credentials file.
